extern/nlohmann/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# Detray library, part of the ACTS project (R&D line)
+#
+# (c) 2022 CERN for the benefit of the ACTS project
+#
+# Mozilla Public License Version 2.0
+
+# CMake include(s).
+cmake_minimum_required( VERSION 3.14 )
+include( FetchContent )
+
+# Tell the user what's happening.
+message( STATUS "Building nlohmann as part of the Detray project" )
+
```

Comment:
Looking at the https://gitlab.cern.ch/atlas/atlasexternals/-/blob/master/External/nlohmann_json/CMakeLists.txt and https://github.com/acts-project/acts/blob/main/CMakeLists.txt#L227, you probably also want to add

```cmake
set( JSON_BuildTests FALSE CACHE BOOL
   "Turn the tests in nlohmann_json on/off" )
```

around here.

Reply:
Understood, added.